source/tyr/tests/integration/instance_test.py

```diff
@@ -120,6 +120,8 @@ def test_update_instances(create_instance):
         "max_successive_physical_mode": 3,
         "final_line_filter": True,
         "max_extra_second_pass": 1,
+        "additional_time_after_first_section_taxi": 42,
+        "additional_time_before_last_section_taxi": 789,
```

Comment:
You need to test these attributs as in https://github.com/CanalTP/navitia/blob/a6295cc80546d4f4560ffb6df513ba846172f254/source/tyr/tests/integration/instance_test.py#L351